spec/factories/users.rb

```diff
@@ -4,5 +4,7 @@
 
     first_name { Faker::Name.first_name }
     last_name { Faker::Name.last_name }
+    email { Faker::Internet.email }
```

Comment:
You can add uniqueness here as well: `Faker::Internet.unique.email` (from https://github.com/faker-ruby/faker#ensuring-unique-values).